packages/console/src/components/ItemPreview/index.tsx

```diff
@@ -4,7 +4,7 @@ import * as styles from './index.module.scss';
 
 type Props = {
   title: string;
-  subtitle?: string;
+  subtitle?: string | unknown;
```

Comment:
This looks strange.
Should we limit the translation method to return `string?` value?

Reply:
updated ptal